apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx

```diff
@@ -1,9 +1,60 @@
 import { FileOpenPreference, files, SdkError } from '@microsoft/teams-js';
-import React, { ReactElement } from 'react';
+import React, { ChangeEvent, ReactElement } from 'react';
 
+import { noHostSdkMsg } from '../../App';
 import { ApiWithCheckboxInput, ApiWithoutInput, ApiWithTextInput } from '../utils';
+import { ApiContainer } from '../utils/ApiContainer';
 import { ModuleWrapper } from '../utils/ModuleWrapper';
 
+export const FileUpload: React.FC = () => {
```

Comment:
Does this control call teams-js? It looks like it's just getting files via the HTML input and then displaying the result, is that right?

Reply:
It doesnot. Essentially on any iframe/webview for <input type="file' /> you could choose and select files. This control enables testing of that